mysql-to-kv-delta/mysql-to-kv-delta.py

```diff
@@ -0,0 +1,59 @@
+
+
+      
+
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+import time
+import datetime
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import v3io_frames as v3f
+
+
+def handler(context, event):
+    delta_interval_minute = os.getenv('DELTA_INTERVAL_MINUTE')
```

Comment:
set ENV vars as global variables (example https://github.com/nuclio/nuclio-templates/blob/master/kafka-to-kv/kafka-to-kv.py#L18)